src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyout.xaml.cs

```diff
@@ -147,8 +147,14 @@ private void PositionAdornment()
                 ? _textView.ViewportRight - width
                 : desiredLeft;
 
-            Canvas.SetTop(this, top);
-            Canvas.SetLeft(this, left);
+            MaxWidth = _textView.ViewportRight;
+            if (MinWidth > _textView.ViewportWidth)
+            {
+                MinWidth = _textView.ViewportWidth;
```

Comment:
Should we reset MinWidth to 200 (which I see is set in the XAML) when the viewport width allows it?

Reply:
Indeed!